File: src/main/java/com/sprout/beemod/item/food/ModFoods.java
package com.sprout.beemod.item.food;

import net.minecraft.world.food.FoodProperties;

public class ModFoods {

    public static final FoodProperties SOUL_NECTAR_BOTTLE = new FoodProperties.Builder()
            .nutrition(6)
            .saturationModifier(0.6f)
            .alwaysEdible()
            .build();
}

File: src/main/kotlin/com/sprout/beemod/item/food/ModFoods.kt
package com.sprout.beemod.item.food

import net.minecraft.world.food.FoodProperties

object ModFoods {

    val SOUL_NECTAR_BOTTLE = FoodProperties.Builder()
        .nutrition(6)
        .saturationModifier(0.6f)
        .alwaysEdible()
        .build()

}